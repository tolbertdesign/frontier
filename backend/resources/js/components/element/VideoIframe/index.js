import VideoIframe from './VideoIframe.vue'

export default VideoIframe
