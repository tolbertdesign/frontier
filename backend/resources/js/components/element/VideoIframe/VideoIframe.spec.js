import { shallowMount, createLocalVue } from '@vue/test-utils'
import VideoIframe from './VideoIframe.vue'

const localVue = createLocalVue()

const source = 'vimeo'
const videoId = '123456789'
const color = '111111'
const autoplay = false
const configId = 'JcxcCN5H'

describe('VideoIframe', () => {
  it('Snapshop test for VideoIframe', () => {
    const wrapper = shallowMount(VideoIframe, {
      localVue,
      propsData: {
        source: source,
        videoId: videoId,
        color: color,
        autoplay: autoplay,
        configId: configId,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
