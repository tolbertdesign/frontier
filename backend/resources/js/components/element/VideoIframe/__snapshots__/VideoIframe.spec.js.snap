// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoIframe Snapshop test for VideoIframe 1`] = `
<div><iframe src="https://player.vimeo.com/video/123456789?color=111111" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen="allowfullscreen" style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"></iframe>
  <!---->
</div>
`;
